org.eclipse.scanning.example/src/org/eclipse/scanning/example/malcolm/IEPICSv4Device.java

```diff
@@ -0,0 +1,16 @@
+package org.eclipse.scanning.example.malcolm;
+
+import java.util.Map;
+
+import org.epics.pvdata.pv.PVStructure;
+
+public interface IEPICSv4Device {
+
+	public void start() throws Exception;

```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS00112)